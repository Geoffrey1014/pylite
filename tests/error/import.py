import math

print math.typo
