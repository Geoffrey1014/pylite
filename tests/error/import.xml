lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    load_module c0
    load_this_module
    store_field c0
    load_global c0
    load_field c1
    print
    load_const c2
    return
  const_vector
    math
    typo
    None
  guard_vector
  line_number_map 1 1 1 3 3 3 1 1
