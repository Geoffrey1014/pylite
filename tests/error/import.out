AttributeError:
  At import.py, line 3, in __init__()
